pkg/api/kptfile/v1alpha2/validation.go

```diff
@@ -183,6 +183,20 @@ func IsKRM(n *yaml.RNode) error {
 	return nil
 }
 
+//nolint
```

Comment:
curious what linter is complaining about here ?

Reply:
that the error message ends in punctuation, and that I am ignoring the error of PipeE. IMO if PipeE throws an error we can safely ignore it.